allure-docker-api/templates/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <title>Allure Docker Service</title>
     <link rel="shortcut icon" href="{{ url_for('static', filename='favicon') }}">
-    <meta http-equiv="Refresh" content="0; url=/allure-docker-service/swagger"/>
+    <meta http-equiv="Refresh" content="0; url={{url}}"/>
```

Comment:
This change was required to support loading the swagger ui at the root when using a prefix